### DISCUSSION TITLE
Is PDF Viewer ignoring the text alignment in form fields?

### DISCUSSION THREAD

**Discussion by hagealex** [stephanrauh/ngx-extended-pdf-viewer#1950] [Q&A] [answered]
Hello everyone!

I have a PDF form where specific fields have a text alignment with the value `right`. I've created the form with Adobe Acrobat.

When I fill the form in the browser everything looks good. For example I have this date field here where the text is aligned right:

![image](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/21688684/f5b11b20-9c3d-4a5a-af4a-574820a2f2e9)

When I go into the print preview the text is not aligned right anymore: 

![image](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/21688684/b701b7e8-ed71-4da4-a211-c45069a7e3e8)

When I open the PDF directly in Chrome or Edge it works as expected. When I open the file directly in Firefox I get the same issue.

Is this intended behavior?

**Comment by stephanrauh** [comment, marked as answer]
This sounds like a bug of the base library, pdf.js. Please check if you can reproduce the bug on https://mozilla.github.io/pdf.js/web/viewer.html. If so, please report a bug at https://github.com/mozilla/pdf.js/issues. Just make sure not to mention ngx-extended-pdf-viewer: pdf.js is the PDF viewer of Firefox, and they won't accept any ticket that's not related to Firefox.

**Reply by hagealex** [reply]
Thank you for the hint! I'll check this and report the bug if it also appears in the base lib!
